fairlearn/postprocessing/_constants.py

```diff
@@ -11,6 +13,45 @@
 
 DEMOGRAPHIC_PARITY = "demographic_parity"
 EQUALIZED_ODDS = "equalized_odds"
+ACCURACY_SCORE = "accuracy_score"
+
+def confusion_matrix_summary(*, true_positives, false_positives, true_negatives, false_negatives):
```

Comment:
That's not really a "constant" :-)

Reply:
shall we rename `_constants` -> `_base`?

Reply:
Base kind of makes me think "base class". So far everything like that is named utilities. The constants just don't belong in the same file, though. Can you put the functions/logic in a utilities (or other name) file?

Reply:
i see your point. let me see what i can do. i resist creating a lot of small files...

Reply:
If you find a more suitable name than constants or base I don't mind that either... :-)

Reply:
`extended_confusion_matrix`?

Reply:
I'm moving it elsewhere and renaming to `_extend_confusion_matrix`.